data/entities.json5

```diff
@@ -11273,4 +11273,12 @@
     "domains": ["*.auth0.com"],
     "examples": ["cdn.auth0.com"]
   }
+  {
+    "name": "Jaywing",
+    "homepage": "https://jaywing.com/",
+    "categories": ["marketing"],
+    "domains": ["*.almanac.jaywing.com"],
```

Comment:
I think it's better to just wildcard the top level domain. eg *.jaywing.com

Reply:
updated